sdks/java/core/src/main/java/org/apache/beam/sdk/coders/AvroCoder.java

```diff
@@ -217,6 +217,7 @@ public static CoderProvider getCoderProvider() {
   }
 
   private final Class<T> type;
+  private final boolean useReflection;
```

Comment:
Why not to use the previous name `useReflectApi` ?

Reply:
Well, maybe it's only me, but `reflection` is rather associated with Java Reflection for me. So I think `useReflectApi` reflects :) better the initial intention of this.

Reply:
It is actually based on java reflection. See the package [definition](https://avro.apache.org/docs/current/api/java/org/apache/avro/reflect/package-summary.html)
> Use Java reflection to generate schemas and protocols for existing classes.